src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,66 @@
-export const Pagination = () => {};
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const currentPageMinus = currentPage === 1 ? currentPage : currentPage - 1;
+  const currentPagePlus = currentPage === total ? currentPage : currentPage + 1;
+  const isFirstPage = currentPage === 1;
+  const isLastPage = currentPage === total;
+
+  return (
+    <>
+      <ul className="pagination">
+        <li className={`page-item ${isFirstPage && 'disabled'}`}>
```

Comment:
Use classnames library for conditional setting classes https://www.npmjs.com/package/classnames. As in your case you set classes `false` if a condition is false
![image](https://github.com/mate-academy/react_pagination/assets/63243136/bb2b4014-cd0e-4fb7-a783-4f4b2643800e)